benchmark.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
```

Comment:
This should probably auto-switch the directory and/or check if it's in the right one, especially since we're rm -rf-ing stuff

Reply:
Something like that could help (changes directory to the script location):
```
cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")"
```